index.html

```diff
@@ -994,6 +994,79 @@
 Issue(16): This description of what makes information sensitive still needs to
 be refined.
 
+## Unexpected profiling {#hl-unexpected-profiling}
+
+Contributes to [=surveillance=], [=correlation=], [=identification=], and
+singling-out / discrimination.
+
+Unexpected profiling occurs when a site is able to learn attributes or
+characteristics about a person, that a) the site visitor did not intend the site
+to learn, and b) the site visitor reasonably could not anticipate a
+site would be able to learn.
+
+Profiling contributes to, but is distinct from, other privacy risks discussed
+in this document.
+For example, unexpected profiling may contribute to
+[[[#hl-recognition-same-site]]], by adding stable and semi-identifying
+information that can contribute to <a
+data-cite="fingerprinting-guidance#dfn-browser-fingerprinting">browser
+fingerprinting</a>. Unexpected profiling is distinct from same-site
+recognition though, in that a person may wish to not share some kinds
+of information about themselves <em>even in the presence of guarantees
+that such information will not lead to them being re-identified</em>.
+
+Similarly, unexpected profiling is related to [[[#hl-sensitive-information]]],
+but the former is a superset of the latter;
+all cases of unexpected sensitive information disclosure are examples of
+unexpected profiling, but Web users may have attributes or characteristics
+about themselves that are not universally thought of as "sensitive", but which
+they never the less do not wish to share with the sites they visit. People
+may wish to not share these "non-sensitive" characteristics
+for a variety of reasons (e.g., a person may worry that their ideas of what
+counts as "sensitive" is different from others, a person might might be ashamed
+or uncomfortable about a character trait or they might simply not wish to be profiled).
+
+Profiling occurs for many reasons. It can be used to facilitate price
+discrimination or offer manipulation, to make inferences about what products or
+services users might be more likely to purchase, or more generally, for a site
+to learn attributes about the Web user the Web user does not intend to share.
+Unexpected profiling can also contribute to feelings of powerlessness and
```

Comment:
Is there a citation for this that we could add here? Please note: this is not a challenge as I comletely agree. And I think it would be stronger with a citation...

Reply:
If folks think a citation is needed here, let me reach out to folks who do research in this area to see what they think would be most on point. I've created an issue to capture this, but I'd like to merge this PR in the meantime https://github.com/w3ctag/privacy-principles/issues/66